posix.mak

```diff
@@ -733,12 +733,25 @@ LOOSE_CHANGELOG_FILES:=$(wildcard $(DMD_DIR)/changelog/*.dd) \
 changelog/next-version: ${DMD_DIR}/VERSION
 	$(eval NEXT_VERSION:=$(shell changelog/next_version.sh ${DMD_DIR}/VERSION))
 
-changelog/pending.dd: changelog/next-version | ${STABLE_DMD} ../tools ../installer
-	[ -f changelog/pending.dd ] || $(STABLE_RDMD) $(TOOLS_DIR)/changed.d \
-		$(CHANGELOG_VERSION_LATEST) -o changelog/pending.dd --version "${NEXT_VERSION}" \
-		--date "To be released" --nightly
+changelog/pending.dd: $(TOOLS_DIR)/contributors.d $(TOOLS_DIR)/changed.d $(CHANGELOG_FILES) changelog/next-version | ${STABLE_DMD} $(TOOLS_DIR) $(INSTALLER_DIR)
```

Comment:
```
make: *** No rule to make target 'changelog/2.033', needed by 'changelog/pending.dd'.  Stop.
```

`$(CHANGELOG_FILES)` is not suitable as a target/dependency name directly, because it does not contain file extensions (they are being stripped by `basename`).

Reply:
Oh that error is due to Martin renaming it one day ago (-> https://github.com/dlang/dlang.org/pull/1953/files) and myself forgetting about it in the rebase.